lxd/storage/backend_lxd.go

```diff
@@ -502,21 +502,18 @@ func (b *lxdBackend) CreateInstanceFromCopy(inst instance.Instance, src instance
 
 	contentType := InstanceContentType(inst)
 
-	if b.driver.HasVolume(volType, project.Prefix(inst.Project(), inst.Name())) {
-		return fmt.Errorf("Cannot create volume, already exists on target")
-	}
-
 	// Get the root disk device config.
 	_, rootDiskConf, err := shared.GetRootDiskDevice(inst.ExpandedDevices().CloneNative())
 	if err != nil {
 		return err
 	}
 
-	// Get the volume name on storage.
-	volStorageName := project.Prefix(inst.Project(), inst.Name())
+	vol := b.newVolume(volType, contentType, project.Prefix(inst.Project(), inst.Name()),
```

Comment:
@monstermunchkin please dont overflow the lines like this, it makes grepping for code patterns more difficult.